spring-data-microservice-transaction-sample/README.md

```diff
@@ -112,44 +112,39 @@ The Entity Relationship Diagram for the schema is as follows:
-3. Getting order information by order ID. It is a transaction in the Order Service
-4. Getting order information by customer ID. It is a transaction in the Order Service
-5. Repayment (reduces the amount in the `credit_total`). It is a transaction in the Customer Service.
-
-### Service Endpoints
+### Service endpoints
 
 The endpoints defined in the services are as follows:
 
-Customer Service:
-
-- getCustomerInfo
-- payment
-- prepare
-- validate
-- commit
-- rollback
-- repayment
-
-Order Service:
-
-- placeOrder
-- getOrder
-- getOrders
-
-The `getCustomerInfo` endpoint of the Customer Service is for transaction #1 (Getting customer information).
-
-And the `placeOrder` endpoint of the Order Service and the `payment`, `prepare`, `validate`, `commit`, and `rollback` endpoints of the Customer Service are for transaction #2 (Placing an order) that spans the Order Service and the Customer Service.
-The Order Service starts the transaction with the `placeOrder` endpoint, which calls the `payment`, `prepare`, `validate`, `commit`, and `rollback` endpoints of the Customer Service.
-
-The `getOrder` of the Order Service is for transaction #3, and The `getOrders` of the Order Service is for transaction #4.
-
-And the `repayment` endpoint of the Customer Service is for transaction #5.
+- Customer Service
+  - `getCustomerInfo`
+  - `getCustomerInfoForTwoPhaseCommit`
+  - `payment`
+  - `prepare`
+  - `validate`
+  - `commit`
+  - `rollback`
+  - `repayment`
+
+- Order Service
+  - `placeOrder`
+  - `getOrder`
+  - `getOrders`
+
+### What you can do in this sample application
+
+The sample application supports the following types of transactions:
+
+- Get customer information through the `getCustomerInfo` endpoint of the Customer Service.
+- Place an order by using a line of credit through the `placeOrder` endpoint of the Order Service and the `payment`, `prepare`, `validate`, `commit`, and `rollback` endpoints of the Customer Service.
+  - Checks if the cost of the order is below the customer's credit limit.
+  - If the check passes, records the order history and updates the amount the customer has spent.
+- Get order information by order ID through the `getOrder` endpoint of the Order Service and the `getCustomerInfoForTwoPhaseCommit`, `prepare`, `validate`, `commit`, and `rollback` endpoints of the Customer Service.
+- Get order information by customer ID through the `getOrders` endpoint of the Order Service and the `getCustomerInfoForTwoPhaseCommit`, `prepare`, `validate`, `commit`, and `rollback` endpoints of the Customer Service.
+- Make a payment through the `repayment` endpoint of the Customer Service.
+  - Reduces the amount the customer has spent.
+
+Merging the `getCustomerInfoForTwoPhaseCommit` endpoint into the `getCustomerInfo` endpoint may be a good way to reduce the number of endpoints in the Customer Service. We intentionally separated the endpoints to keep them simple as a sample application.
```

Comment:
I'd like to put this part into a note section and avoid using the first-person narrative (`we`). How about we change this as follows?:

```suggestion
{% capture notice--info %}
**Note**

Merging the `getCustomerInfoForTwoPhaseCommit` endpoint into the `getCustomerInfo` endpoint may be a good way to reduce the number of endpoints in the Customer Service. The endpoints are intentionally separated for the sake of simplifying them in the sample application.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Reply:
@josh-wong Thanks! I'll merge your suggestions. BTW, we are discussing whether to merge `getCustomerInfoForTwoPhaseCommit` into `getCustomerInfo` and this note might be removed later 😅 

> avoid using the first-person narrative (we)

Just out of curiosity, I sometimes see `you` to represent the readers of the document. `you` is okay while `we` should be avoided?

Reply:
> BTW, we are discussing whether to merge getCustomerInfoForTwoPhaseCommit into getCustomerInfo and this note might be removed later 😅

Sorry, I should have upscrolled to understand that part in more detail😅

> Just out of curiosity, I sometimes see `you` to represent the readers of the document. `you` is okay while `we` should be avoided?

Correct. In documentation, using `you` is preferred when referring to the reader. Using `we` is a little indirect since readers might be confused on if `we` means `Scalar` or if `we` is an indirect way of saying `you`. In this case, we can use passive voice to avoid an awkward sentence construction, like `Scalar intentionally separated ...`.

Reply:
> Sorry, I should have upscrolled to understand that part in more detail😅

Oh, you don't need to say sorry. Your suggestion apparently improves the current PR and I really appreciate it.

> Using we is a little indirect since readers might be confused on if we means Scalar or if we is an indirect way of saying you

It really makes sense... Thanks for the explanation!